core/vm/contracts.go

```diff
@@ -74,6 +79,9 @@ var PrecompiledContractsIstanbul = map[common.Address]PrecompiledContract{
 	common.BytesToAddress([]byte{7}): &bn256ScalarMulIstanbul{},
 	common.BytesToAddress([]byte{8}): &bn256PairingIstanbul{},
 	common.BytesToAddress([]byte{9}): &blake2F{},
+
+	common.BytesToAddress([]byte{102}): &sha3fip{},
```

Comment:
since we already have vrf at 255. How about going backward from there like 254 and 253?

Reply:
@rlan35  changed the address as suggested.

Reply:
Please add a new PrecompiledContractsXXXXX   and add a forking epoch to enable it. You can follow the code here: https://github.com/harmony-one/harmony/pull/3703/files

Reply:
hi @rlan35, thanks for the reference code, much helpful,
Can I also please ask in this file 
[https://github.com/harmony-one/harmony/blob/b34d06b2690395ca0ab64c34be72484ed92249d0/internal/params/config.go#L62](url)
what would be the epoch number for the MainnetChainConfig & TestnetChainConfig and for the rest of the configs I assume it is Zero(0).

Reply:
Sorry, missed the thread. You can put "EpochTBD" for mainnet and testnet for now. 0 for other configs. We will decide a number before release.